DTS-STN/dts-rmp
# Style error

# Description

Added styling for the 404 error as per Pond's design request. 
The page has both english and french on the same page. The strings are hardcoded. 
Button links to the home page. 

## Test Instructions

launch app to a page that does not exist. 
notice that the text for english and french. 
click the button to see the home page. 

## Help Requested
the linter insisted on make changes to AppNavButton and AppNavHeader. Please take a look and see if the linter made the correct change. 

## Checklist
- [x ] E2E tests have been added/updated
